jenkins-scripts/dsl/brew_release.dsl

```diff
@@ -191,6 +169,25 @@ bottle_job_builder.with
      project / 'properties' / 'hudson.plugins.copyartifact.CopyArtifactPermissionProperty' / 'projectNameList' {
       'string' "${bottle_hash_updater_job_name}"
      }
+     project  / triggers / 'org.jenkinsci.plugins.ghprb.GhprbTrigger' {
+         adminlist 'osrf-jenkins j-rivero scpeters'
+         orgslist 'osrf'
+         useGitHubHooks(true)
+         allowMembersOfWhitelistedOrgsAsAdmin(false)
```

Comment:
Don't want to share admin rights with the `osrf` organization? :)

Reply:
yeah, I meant to fix that. I had been thinking that there are lots of members of `osrf` outside the ignition team, so I set that to false, but what I had intended to do was to use `ignitionrobotics` as the relevant org, which I've done in 939619791b24818b07d00a06d26a97944d54450b